src/main/resources/templates/newuserform.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<title>User Managment system</title>
+<div th:replace="fragmentes/header :: header-css" />
+</head>
+<body>
+<div th:replace="fragmentes/header :: header" />
+<div>
+
+<div class="container">
+	<div class="row">
+		<div class="col-md-6 col-md-offset-3">
+			<div class="panel panel-default">
```

Comment:
I highly recommend using string resources to show constant strings e.g 'Add New User' on the web page.
Remove all the constant strings and replace them with the property name on the properties files.